extension/snippets/xcomponent.json

```diff
@@ -2,10 +2,19 @@
 	"Create a XComponent state": {
 		"prefix": "xcstate",
```

Comment:
could we stop putting xc everywhere?

@nicolaserny

Reply:
I think it is a good idea for snippets and a common practice.
You can easily find the xcomponent snippets with intellisense by typing xc...

Reply:
Ok :)